modules/Microsoft.Synapse/workspaces/.bicep/nested_cmkRbac.bicep

```diff
@@ -9,7 +9,7 @@ resource keyVault 'Microsoft.KeyVault/vaults@2019-09-01' existing = {
 
 // Assign RBAC role Key Vault Crypto User
 resource workspace_cmk_rbac 'Microsoft.Authorization/roleAssignments@2022-04-01' = if (usesRbacAuthorization) {
-  name: '${workspaceIdentity}-cmk-rbac'
+  name: guid('${workspaceIdentity}-cmk-rbac')
```

Comment:
Also, I know it's not part of the intended change, but could you rename `workspaceIdentity` to `workspaceIndentityPrincipalId`?

Reply:
Done